frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,95 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "aut-user-preferences"
+              And user is at Add page
+
+
+        @smoke @to-do
+        Scenario: Access user preference from the masthead: UP-01-TC01
+             When user clicks on user dropdown
+             Then user will see User Preference
+
+
+        @regression @to-do
+        Scenario: Setting Developer preference for perspective: UP-01-TC02
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user clicks on Perspective dropdown on User Preferences page
+              And user selects "Developer" option
+             Then user can see "Developer" option selected in the perspective switcher
+
+
+        @regression @to-do
+        Scenario: Setting a preference for a project: UP-01-TC03
+            Given user has created project "test-preference1"
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user clicks on Project dropdown on User Preferences page
+              And user searches project "test-preference1"
+              And user selects the project "test-preference1" from the dropdown
+              And user clicks on Add from navigation menu
+             Then user can see "test-preference1" option selected in project namespace
+
+
+        @regression @to-do
+        Scenario: Creating project with project preference: UP-01-TC04
+            Given user has created project "test-preference1"
+              And user is at Add page
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user clicks on Project dropdown on User Preferences page
+              And user types project "test-preference2" in search bar
+              And user clicks on the Create project option from the dropdown
+              And user clicks on Add from navigation menu
```

Comment:
same as above

Reply:
As per my understanding of the epic, This should be true if the active project when the user was initially on the Add page, was `ALL_PROJECTS`. 
but generally, setting a preference for namespace means, this preference will be honored when the console loads without any url params. in other cases, if an active project is detected, it will have preference over the preferred project.
cc @jerolimov

Reply:
This is outdated. But yep, when the user switches to the add page she or he should see the right namespace. But the console saves the latest namespace internally, so maybe a page reload is required.

Reply:
Updated